packages/location-field/src/styled.tsx

```diff
@@ -74,26 +74,26 @@ export const Dropdown = ({
         aria-expanded={open}
         aria-label={intl.formatMessage(
           {
-            id: "otpUi.LocationField.suggestedLocationsLong",
             defaultMessage: "Open the list of location suggestions",
             description:
-              "Text to show as a a11y label for the button that opens the dropdown list of locations"
+              "Text to show as a a11y label for the button that opens the dropdown list of locations",
+            id: "otpUi.LocationField.suggestedLocationsLong"
           },
           { locationType }
```

Comment:
Passing parameters is no longer needed.

Reply:
Oh thanks for catching that totally missed that in all the stress!

Reply:
Remove the `locationType` prop from the `DropDown` component.